.github/workflows/build-contracts-and-push-to-r2.yaml

```diff
@@ -22,6 +22,29 @@ jobs:
       packages: write
       id-token: write
     steps:
+      - name: Check for release information if present
+        id: check-release
+        run: |
+          commit_message="${{ github.event.head_commit.message }}"
```

Comment:
This is the template for the release commits:
`"chore: release {{crate_name}} {{version}}"`
and it is automatic so it will always be like this. Do you prefer we change how we are currently looking for release commits in this PR?

Reply:
Added.